server_http.hpp

```diff
@@ -320,6 +321,24 @@ namespace SimpleWeb {
                     request->method=line.substr(0, method_end);
                     request->path=line.substr(method_end+1, path_end-method_end-1);
 
+					//search and populte query_string
+					size_t qs_start;
+					if ((qs_start = request->path.find('?')) != std::string::npos)
+					{
```

Comment:
I think this would be a bit cleaner:
`auto qs_start = request->path.find('?');`
`if (qs_start != std::string::npos)`
`{`